pandas/tests/dtypes/test_dtypes.py

```diff
@@ -414,25 +414,22 @@ def test_construction_from_string(self, dtype):
         assert is_dtype_equal(dtype, result)
         result = PeriodDtype.construct_from_string("period[D]")
         assert is_dtype_equal(dtype, result)
+        msg = "Cannot construct a 'PeriodDtype' from "
```

Comment:
can you parameterize these (like they were); also can add whats on line 430 in the same test

Reply:
That was unfortunately a change made in a different pull request and my ide keeps bringing those commits with each new branch I make. I had added the parameterized checks in the beginning, so I have pasted that code back in. Unfortunately for that last check, it has a different type of error message that wouldn't be encapsulated with the error message and it wouldn't fit the parameterized pytest check datatype as everything else is of type string.